Softeq.XToolkit.WhiteLabel/Validation/Rules/IsNotNullOrEmptyRule.cs

```diff
@@ -0,0 +1,20 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+namespace Softeq.XToolkit.WhiteLabel.Validation.Rules
+{
+    public class IsNotNullOrEmptyRule : IValidationRule<string>
```

Comment:
Maybe it would be better to name it "IsStringEmpty" or something like that? (to mark, that this will only work with Strings). Also, it's strange to see "IsNotNullOrEmpty" in class name and "IsNullOrWhiteSpace" inside the Check method

Reply:
Renamed to `StringNotEmptyRule`